src/uucore/src/lib/features/quoting_style.rs

```diff
@@ -341,6 +357,14 @@ pub fn escape_name(name: &OsStr, style: &QuotingStyle) -> String {
     }
 }
 
+pub fn escape_name(name: &OsStr, style: &QuotingStyle) -> String {
```

Comment:
why these functions and call them correctly ? :)

Reply:
The main reason was to keep the function signature of `escape_name()` which is used in many places, and I didn't want to impact the rest of the codebase with this change.

I think it's  better in terms of readability for other references to `escape_name()` to not be bothered by a trailing boolean which would be false most of the time.

If some performance concerns arise, I guess we could `#[inline]` them, though I don't think these have a tremendous impact on performance anyway.

In any case, it's not a strong requirement for me. I can easily revert to a single `escape_name()` function if you feel this better suits the codebase.

Reply:
ok, maybe document the why then ?